proxy-node-shared/src/main/java/uk/gov/ida/notification/validations/DestinationUriStringValidator.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.ida.notification.validations;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.net.URI;
+
+public class DestinationUriStringValidator implements ConstraintValidator<ValidDestinationUriString, String> {
+
+    @Override
+    public void initialize(ValidDestinationUriString constraint) { /* intentionally blank */ }
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        if (StringUtils.isBlank(value)) { return true; } // @NotNull should detect nulls
```

Comment:
Feels like it should be returning false.

Reply:
Ah, because my intent (and comment) hasn't been clear enough. This validator shouldn't respond false on a nullable parameter that is null. @NotNull is for preventing null values.